lib/model/subject.dart

```diff
@@ -4,74 +4,46 @@ import 'package:easy_study/model/exam_type.dart';
 import 'package:easy_study/model/priority.dart';
 
 class Subject {
-  String _title, _room, _description;
-  int _id, _timeSpent = 0, _hoursWeek;
-  ExamType _type;
-  Priority _priority;
-  Color _color = Color.fromARGB(255, 0, 0, 0);
+  String title, room, description;
```

Comment:
why are all field variables public now? i would leave them private

Reply:
Because in flutter you only declare variable private if you want to override the getter and setter.